docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -616,6 +617,12 @@ Size of /dev/shm.
 
 This is equivalent to the Podman `--shm-size` option and generally has the form `number[unit]`
 
+### `StopTimeout=`
+
+Seconds to wait before forcibly stopping the container.
+
+This is equivalent to the `podman rm` `--time` option
```

Comment:
Maybe worth to mention that it should be set lower than the actual systemd unit timeout to make sure the podman rm command is not killed by systemd.

Reply:
Done